opentok/opentok.py

```diff
@@ -1309,7 +1311,8 @@ def start_broadcast(self, session_id, options, stream_mode=BroadcastStreamModes.
             If you include RTMP streaming, you can specify up to five target RTMP streams. For
             each RTMP stream, specify 'serverUrl' (the RTMP server URL), 'streamName' (the stream
             name, such as the YouTube Live stream name or the Facebook stream key), and
-            (optionally) 'id' (a unique ID for the stream)
+            (optionally) 'id' (a unique ID for the stream). You can optionally specify lowLatency or
+            DVR mode, but these options are mutually exclusive.
```

Comment:
It is not clear to me from these docs what the `outputs` object looks like. Maybe we should include a code example:

```            "outputs": {
                "hls": {
                    "lowLatency": True,
                    "dvr": True
                    },
                "rtmp": [
                    {
                        "id": "foo",
                        "serverUrl": "rtmp://myfooserver/myfooapp",
                        "streamName": "myfoostream",
                    },
                    {
                        "id": "bar",
                        "serverUrl": "rtmp://mybarserver/mybarapp",
                        "streamName": "mybarstream",
                    },
                ],
            },
```
For a docs example, see https://github.com/opentok/Opentok-Python-SDK/blob/main/opentok/opentok.py#L1117-L1120.

Reply:
Have added this now